gsy_framework/sim_results/electric_blue/all_results.py

```diff
@@ -0,0 +1,103 @@
+from collections import defaultdict
+from typing import Dict
+
+from pendulum import from_format, DateTime, Duration
+
+from gsy_framework.constants_limits import DATE_TIME_FORMAT
+from gsy_framework.enums import AvailableMarketTypes
+from gsy_framework.forward_markets.forward_profile import ALLOWED_MARKET_TYPES
+from gsy_framework.sim_results.electric_blue.aggregate_results import (handle_forward_results,
+                                                                       ForwardDeviceStats,
+                                                                       MARKET_RESOLUTIONS)
+from gsy_framework.sim_results.electric_blue.time_series import ForwardDeviceTimeSeries
+
+
+class ForwardResultsHandler:
+    """Calculate all results for each market slot for forward markets."""
+
+    def __init__(self, slot_length: Duration):
+        self.forward_market_enabled = True
+        self.slot_length = slot_length
```

Comment:
This does not seem to be used in this class. Is it a placeholder for later?
If this is removed from here, it will also make the calling code on gsy-web side a little cleaner IMO.

Reply:
Let's see how [this](https://github.com/gridsingularity/gsy-web/pull/1804#discussion_r995541361) discussion evolves. We may not need it.

Reply:
98d31ba846e6a411599492c96f834a50cb51bbd2

Reply:
https://github.com/gridsingularity/gsy-web/pull/1804#discussion_r1000723049